projects/WeTek_Hub/filesystem/usr/lib/systemd/system/serial-console.service

```diff
@@ -0,0 +1,23 @@
+[Unit]
+Description=Debug Shell on /dev/ttyS0
+DefaultDependencies=no
+ConditionKernelCommandLine=console=ttyS0,115200
+
+[Service]
+WorkingDirectory=/storage
+Environment="ENV=/etc/profile"
+ExecStartPre=/bin/sh -c 'echo -en "\033[?25h"'
+ExecStart=/bin/sh
+Restart=always
+RestartSec=0
+StandardInput=tty
+TTYPath=/dev/ttyS0
+TTYReset=yes
+TTYVHangup=yes
```

Comment:
keep it simple, drop TTYReset and TTYVHangup if you dont *really* need em. there some unwanted side effects may come by using reset / hangup.